### ISSUE TITLE
Getting lower results

### ISSUE THREAD

**Issue by ophirazulai** [JJBOY/BMN-Boundary-Matching-Network#4]
Hi,

Dowloaded your features, trained and evaluated you model.
I get the following results

train subset video numbers: 9649                                                                                                            
validation subset video numbers: 4728                                                                                                       
BMN training loss(epoch 0): tem_loss: 1.152, pem_loss: 0.267, total_loss: 1.419                                                             
BMN testing loss(epoch 0): tem_loss: 1.184, pem_loss: 0.261, total_loss: 1.445                                                              
BMN training loss(epoch 1): tem_loss: 1.095, pem_loss: 0.235, total_loss: 1.329                                                             
BMN testing loss(epoch 1): tem_loss: 1.115, pem_loss: 0.231, total_loss: 1.346                                                              
BMN training loss(epoch 2): tem_loss: 1.070, pem_loss: 0.223, total_loss: 1.293                                                             
BMN testing loss(epoch 2): tem_loss: 1.098, pem_loss: 0.249, total_loss: 1.347                                                              
BMN training loss(epoch 3): tem_loss: 1.056, pem_loss: 0.218, total_loss: 1.274                                                             
BMN testing loss(epoch 3): tem_loss: 1.103, pem_loss: 0.222, total_loss: 1.325                                                              
BMN training loss(epoch 4): tem_loss: 1.042, pem_loss: 0.213, total_loss: 1.255                                                             
BMN testing loss(epoch 4): tem_loss: 1.098, pem_loss: 0.225, total_loss: 1.323                                                              
BMN training loss(epoch 5): tem_loss: 1.028, pem_loss: 0.210, total_loss: 1.238                                                             
BMN testing loss(epoch 5): tem_loss: 1.116, pem_loss: 0.260, total_loss: 1.377
BMN training loss(epoch 6): tem_loss: 1.016, pem_loss: 0.212, total_loss: 1.229
BMN testing loss(epoch 6): tem_loss: 1.101, pem_loss: 0.223, total_loss: 1.324
BMN training loss(epoch 7): tem_loss: 0.999, pem_loss: 0.205, total_loss: 1.204
BMN testing loss(epoch 7): tem_loss: 1.150, pem_loss: 0.222, total_loss: 1.372
BMN training loss(epoch 8): tem_loss: 0.991, pem_loss: 0.204, total_loss: 1.195
BMN testing loss(epoch 8): tem_loss: 1.138, pem_loss: 0.221, total_loss: 1.359
BMN training loss(epoch 9): tem_loss: 0.926, pem_loss: 0.191, total_loss: 1.117
BMN testing loss(epoch 9): tem_loss: 1.163, pem_loss: 0.216, total_loss: 1.379
BMN training loss(epoch 10): tem_loss: 0.901, pem_loss: 0.188, total_loss: 1.089
BMN testing loss(epoch 10): tem_loss: 1.176, pem_loss: 0.215, total_loss: 1.391
BMN training loss(epoch 11): tem_loss: 0.886, pem_loss: 0.187, total_loss: 1.073
BMN testing loss(epoch 11): tem_loss: 1.218, pem_loss: 0.216, total_loss: 1.434
BMN training loss(epoch 12): tem_loss: 0.873, pem_loss: 0.185, total_loss: 1.059
BMN testing loss(epoch 12): tem_loss: 1.237, pem_loss: 0.216, total_loss: 1.453
BMN training loss(epoch 13): tem_loss: 0.863, pem_loss: 0.185, total_loss: 1.048
BMN testing loss(epoch 13): tem_loss: 1.265, pem_loss: 0.217, total_loss: 1.482
BMN training loss(epoch 14): tem_loss: 0.853, pem_loss: 0.183, total_loss: 1.037
BMN testing loss(epoch 14): tem_loss: 1.257, pem_loss: 0.217, total_loss: 1.474

[INIT] Loaded annotations from validation subset.
        Number of ground truth instances: 7293
        Number of proposals: 472632
        Fixed threshold for tiou score: [0.5  0.55 0.6  0.65 0.7  0.75 0.8  0.85 0.9  0.95]
[RESULTS] Performance on ActivityNet proposal task.
        Area Under the AR vs AN curve: 63.50907719731249%
       
Adding an axes using the same arguments as a previous axes currently reuses the earlier instance.  In a future version, a new instance will
always be created and returned.  Meanwhile, this warning can be suppressed, and the future behavior ensured, by passing a unique label to each axes instance.
  "Adding an axes using the same arguments as a previous axes "
AR@1 is          0.31546688605512135
AR@5 is          0.43129027834910183
AR@10 is         0.5023584258878377
AR@100 is        0.7298642533936651

In your paper the Area Under the AR vs AN curve is 67.10 and I'm getting 63.5
and your AR@100 is 75.01 and I'm getting 72.9.

I didn't change your configuration.
Any idea why I'm getting lower results.

Thanks
Ophir

**Comment by JJBOY** [comment]
Hello ， I am not the author .  I am sorry to make you confused.
In my released code. the hyperparameters may are not the optimal choices, I dont save these parameters and the model, you may need to do more expriments to get a good result. In my memery, the best result of my expriments is near to the BSN(74.5%) but could't the get results as the paper.
